### PR TITLE
4.14 RN Bugs for Cloud Compute

### PR DESCRIPTION
**Pull request by cbippley**
Version(s):
4.14

Issue:
[Bug Batching for 4.14 GA](https://issues.redhat.com/browse/OSDOCS-5987)

Link to docs preview:
[Preview](https://66210--docspreview.netlify.app/openshift-enterprise/latest/release_notes/ocp-4-14-release-notes#ocp-4-14-bug-fixes)

Additional information:
[Cloud compute bug filter](https://issues.redhat.com/secure/Dashboard.jspa?selectPageId=12347491#SIGwKWmOqDAaIiGh1DSCAiYg6oaN4RY8HF6AMAwlQMHEVU1ccdWpPFMCNc1SXIGgGCNeaPAtG0HT+ogCErGsyGEN+BWVIiiIKPs54OktK2uutTXLQw21YBte3QawILgh+5JftSCgaB47qFfMcT8UaPCIpglTgdQGgaM1dQLIU7oxHCt27BaQlWEAA)

### CODE REVIEW

File: release_notes/ocp-4-14-release-notes.adoc [hunk 1]
```diff
@@ -1102,6 +1102,14 @@ Targeting {product-title} 4.15, the OpenShift SDN CNI network plugin will not be
 [id="ocp-4-14-cloud-compute-bug-fixes"]
 ==== Cloud Compute
 
+* Previously, the Machine API controller could not determine the zone of machines in vSphere clusters that use multiple zones. With this release, the zone lookup logic is based on the host of a VM and, as a result, machine objects indicate proper zones. (link:https://issues.redhat.com/browse/OCPBUGS-7249[*OCPBUGS-7249*]) 
+
+* Previously, the OpenStack machine API provider had to be restarted to pick up new cloud credentials after a rotation of cloud credentials in the `clouds.yaml` file. As a result, the ability of a machine set to scale to zero could be affected. With this change, cloud credentials are no longer cached, and the provider reads the corresponding secret freshly as needed. (link:https://issues.redhat.com/browse/OCPBUGS-8687[*OCPBUGS-8687*])
```
Comment (mburke5678) [inline release_notes/ocp-4-14-release-notes.adoc:1107]:
```suggestion
* Previously, after the rotation of cloud credentials in the `clouds.yaml` file, the OpenStack machine API provider would need to be restarted in order to pick up the new cloud credentials. As a result, the ability of a machine set to scale to zero could be affected. With this change, cloud credentials are no longer cached, and the provider reads the corresponding secret freshly as needed. (link:https://issues.redhat.com/browse/OCPBUGS-8687[*OCPBUGS-8687*]) 
```

**Comment by mburke5678** [comment]
One nit. Otherwise LGTM

File: release_notes/ocp-4-14-release-notes.adoc (hunk 1 above)
Comment (cbippley) [reply release_notes/ocp-4-14-release-notes.adoc]: Thanks!

**Review by opayne1 — COMMENTED** [review]
/lgtm

**Pull request merged**